app_dart/lib/src/request_handlers/get_release_branches.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:github/github.dart';
+
+import '../../cocoon_service.dart';
+
+/// Return a list of commit shas of the latest 5 branches for google3 roll, beta, and stable.
+///
+/// Branches are sorted based on the version number.
+///
+/// GET: /api/public/get-release-branches
+///
+/// Response: Status 200 OK
+/// {
```

Comment:
The expected response is:
```json
[
{
  "branch": "flutter-2.13-candidate.0",
  "name": "stable"
},
{
  "branch": "flutter-3.2-candidate.5",
  "name": "beta"
},
{
  "branch": "flutter-3.4-candidate.5",
  "name": "dev"
}
]
```

Reply:
Ouch, this format would be way easier to implement. I have now removed the tag and version logics and adapted to this one. Thanks for clarification!